### ISSUE TITLE
Premature patella ossification

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11639]
**Preferred term label:**
Premature patella ossification
feature of CAPS (NLRP3).

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001540
- label: Premature patella ossification
- parents: [Abnormal patella morphology (HP:0003045)](https://hpo.jax.org/app/browse/term/HP:0003045)
- pmid: 
- def: Ossification of the patella is detected earlier than the age range typical for the general population.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11639
- synonyms:
